bazel/copts.bzl

```diff
@@ -57,7 +57,104 @@ GRPC_LLVM_WARNING_FLAGS = [
     "-Wno-unused-function",
 ]
 
+GRPC_LLVM_WINDOWS_WARNING_FLAGS = GRPC_LLVM_WARNING_FLAGS + [
```

Comment:
I guess those are necessarily added but wonder why windows happened to have to have all those exceptions despite the fact that both Linux and Windows use the same clang?

Reply:
clang-cl aims to be MSVC-compatible, and I think `clang-cl -Wall` is translated roughly to `clang -Weverything`, because of the way MSVC's `cl.exe /Wall` works.

https://clang.llvm.org/docs/UsersManual.html#clang-cl:~:text=/Wall%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20Enable%20%2DWeverything

We could structure these lists differently, but I'm not sure it'd be much better since the diagnostic flags are not identical between clang-cl and clang.

Reply:
I see. Thank you for the context. I don't worry too much about this lengthy list as long as we don't have to massage this often over time when changing the version of msvc or clang on Windows.